nanomq_cli/client.c

```diff
@@ -1744,6 +1745,10 @@ static int
 quic_msg_recv_cb(void *rmsg, void *arg)
 {
 	nng_msg *msg = rmsg;
+	if (msg == NULL) {
+		console("%s: Got null msg\n", __func__);
```

Comment:
Plz add more details about the reason why a null msg be got.

Reply:
The max length of `nni_lmq` is enough, it's less likely to print this message...

Reply:
Make the probable reasons printed on to the console

Reply:
stand with wangha!